pom.xml

```diff
@@ -319,6 +320,22 @@
         </configuration>
       </plugin>
 
+      <plugin>
+        <groupId>org.folio</groupId>
+        <artifactId>folio-module-descriptor-validator</artifactId>
+        <version>${folio-module-descriptor-validator.version}</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>validate</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <moduleDescriptroFile>${basedir}/descriptors/ModuleDescriptor-template.json</moduleDescriptroFile>
+        </configuration>
+      </plugin>
+
```

Comment:
I think we could have it only here and not in child poms. You can add 
```
<inherited>false</inherited>
```
to not run it on each submodule.